test/levels_test.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env rspec
+
+require_relative 'test_helper'
+
+module Yast
+  class LevelsTester
+    include Yast::I18n
```

Comment:
in general better to have full client here that include more, so even if security/levels.rb use some shortcuts it will be there so clas `LevelsTester < Client`